addons/website_sale_wishlist/static/src/js/website_sale_tour_wishlist.js

```diff
@@ -47,14 +47,9 @@ tour.register('shop_wishlist', {
             },
         },
         {
-            content: "check that logged in and search for Customizable Desk",
-            extra_trigger: "li span:contains('Mitchell Admin')",
-            trigger: 'form input[name="search"]',
-            run: "text Customizable Desk",
-        },
-        {
-            content: "submit search",
-            trigger: 'form:has(input[name="search"]) .oe_search_button',
```

Comment:
Meh ?

Reply:
Customizable Desk is already the first product, so my ':first' hack (see shop_buy_product) doesn't work for this one. Moreover, i think that you never search for the first product displayed, and the search feature is already tested by the other tour.

Reply:
:+1: